CHANGELOG.md

```diff
@@ -1,5 +1,59 @@
+    * Ability for container app template commands to hook into the new revision draft update system [#423](https://github.com/microsoft/vscode-azurecontainerapps/pull/423)
+    * Display a user setting controlled deployment pop-up when executing revision draft commands [#478](https://github.com/microsoft/vscode-azurecontainerapps/pull/478)
+
+* Log streaming support [#350](https://github.com/microsoft/vscode-azurecontainerapps/pull/350)
+* Deploying to a container app via a connected GitHub repository [#353](https://github.com/microsoft/vscode-azurecontainerapps/issues/313)
+* Tree view and CRUD support for container app secrets
+* Delete existing scale rules [#461](https://github.com/microsoft/vscode-azurecontainerapps/pull/461)
+* Improve activity log support for the majority of commands
+
+### Changed
+* Use smart detection to suggest values when deploying from a container registry [#305](https://github.com/microsoft/vscode-azurecontainerapps/pull/305), [#331](https://github.com/microsoft/vscode-azurecontainerapps/pull/331)
+* Revision draft commands:
+    * Reconfigure existing `Scaling` commands and items to utilize the new revision draft design [#420](https://github.com/microsoft/vscode-azurecontainerapps/issues/420)
+        * `Edit Scaling Range` [#466](https://github.com/microsoft/vscode-azurecontainerapps/pull/466)
+        * `Add Scale Rule` [#423](https://github.com/microsoft/vscode-azurecontainerapps/pull/423)
+        * `Delete Scale Rule` [#461](https://github.com/microsoft/vscode-azurecontainerapps/pull/461)
+    * `Update Container Image` was added as the draft alternative to `Deploy to Container App` [#477](https://github.com/microsoft/vscode-azurecontainerapps/pull/477)
+
+### Fixed
+* Remove `Activate`, `Deactivate`, and `Restart` revision commands when not applicable [#356](https://github.com/microsoft/vscode-azurecontainerapps/pull/356)
+* Fix command palette support for `Restart` [#487](https://github.com/microsoft/vscode-azurecontainerapps/pull/487)
+
+### Removed
+* `Deploy to Container App...` was removed in favor of `Deploy Workspace to Container App` or `Update Container Image`
+    * `Deploy Workspace to Container App` can be used if one wishes to immediately deploy a VS Code workspace project OR
+    * `Update Container Image` can be used to update the container app's image in draft mode and then later deployed
+
+### Engineering
+* Decoupled `Ingress` commands from the `createContainerApp` workflow [#371](https://github.com/microsoft/vscode-azurecontainerapps/pull/371), [375](https://github.com/microsoft/vscode-azurecontainerapps/pull/375)
+* Improved telemetry types and handling across all major commands [#514](https://github.com/microsoft/vscode-azurecontainerapps/pull/514), [#517](https://github.com/microsoft/vscode-azurecontainerapps/pull/517), [#524](https://github.com/microsoft/vscode-azurecontainerapps/pull/524)
```

Comment:
Generally we don't normally list telemetry changes in our changelog. I'm in favor of removing it.